src/compas_fab/robots/semantics.py

```diff
@@ -115,6 +119,18 @@ def _get_groups(root, robot_model):
     return groups
 
 
+def _get_group_states(root):
+    group_states = {}
+
+    for group_state in root.findall('group_state'):
+        group_state_name = group_state.attrib['name']
+        group_name = group_state.attrib['group']
+        joint_dict = {joint.attrib['name']: joint.attrib['value'] for joint in group_state.findall('joint')}
+        group_states.setdefault(group_name, {})[group_state_name] = joint_dict
```

Comment:
Group state names are not unique (see panda.srdf), hence the nested dictionaries.  I also thought it would be more user friendly to have the group names as the top level keys rather than the group state names, but maybe others feel differently.

Reply:
@matteo-pacher

Reply:
awesome! should I test in on the `rfl_description`?

Reply:
sure!

Reply:
I tested the PR on `rfl_description` and all seems to work fine: group states can be retrieved properly, their joint names and values match the srdf information.
I wonder if it makes sense to expose the functionality in the `robot.py`, like it happens for `group_names` function.
I guess something like:

```
    @property
    def group_states(self):
        self.ensure_semantics()
        return self.semantics.group_states
```
but also maybe have a simple function to return a full configuration for a group_state?

Reply:
Thanks for testing this out.  Those are good ideas.  I will add the group_states property in this PR, but getting a full configuration from a group_state will be in another issue.

Reply:
you're welcome!
you're right, I see how getting a full configuration from a group_state is maybe not a good idea.
Thank you Beverly!